.goreleaser.yaml

```
@@ -100,6 +100,8 @@ brews:
   - tap:
       owner: ublue-os
       name: homebrew-tap
+      token: "{{ .Env.HOMEBREW_TAP_REPO_TOKEN }}"
```

Does this need to be 

```suggestion
      token: "{{ .Env.TAP_GITHUB_TOKEN }}"
```

?

---

great question, the docs could read that way, let's try it.